getting-started.md

```diff
@@ -491,3 +491,100 @@ $ kubectl get secret folder-outputs
 NAME         TYPE     DATA   AGE
 vm-outputs   Opaque   1      18m
 ```
+
+# For EC
+
+### Apply Provider configuration
+
+```shell
+$ export EC_API_KEY=xxx
```

Comment:
Sorry, I am new to EC. I wonder whether `EC_API_KEY` is commonly known for users of EC. If yes, feel free to keep it as it is, if no, please explain a little bit on how to get the key.

Reply:
I am not certain so I have added some detail about getting a API key from the Elastic Cloud documentation.